cinema_service/settings.py

```
@@ -86,8 +87,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB"),
+        "USER": os.environ.get("POSTGRES_USER"),
```

DB env usage: `NAME`, `USER`, and `PASSWORD` are read directly from environment variables. If any of `POSTGRES_DB`, `POSTGRES_USER`, or `POSTGRES_PASSWORD` are missing from the .env, Django will get `None` here and fail to connect at runtime with a confusing error. Consider either providing safe defaults for local development or adding an entrypoint validation that fails fast with a clear message when required DB env vars are missing. This will make `docker-compose up` easier to debug for reviewers. (Related lines: NAME/USER/PASSWORD are here.)